internal/cli/atlas/privateendpoints/gcp/describe.go

```diff
@@ -26,8 +26,9 @@ import (
 	"github.com/spf13/cobra"
 )
 
-var describeTemplate = `ID	REGION	STATUS	ERROR
-{{.ID}}	{{.RegionName}}	{{.Status}}	{{.ErrorMessage}}
+var describeTemplate = `ID	GROUP NAME	REGION	STATUS	ERROR{{if .EndpointGroupNames}}{{range .EndpointGroupNames}}
+{{$.ID}}	{{.}}	{{$.RegionName}}	{{$.Status}}	{{$.ErrorMessage}}{{end}}{{else}}
+{{$.ID}}	N/A	{{$.RegionName}}	{{$.Status}}	{{$.ErrorMessage}}{{end}}
```

Comment:
[nit/style] Not blocking, just curious. Could we use an inline if else for the N/A check? Something like:
`{{if .}}{{.}} {{else}}N/A{{end}}`

Reply:
since we're looping through `EndpointGroupNames`, if we move the if/else to inline, we would cause a regression in cases where  `EndpointGroupNames` is empty since we would skip the whole line. 


The only workaround I could think of would be to have this if/else inline would be to update the ` Run()` result forcefully with `EnpointGroupNames = ["N/A"]` but that would alter the JSON output as well.

Reply:
Thanks for the detail!